lib/rules/no-warning-comments.js

```diff
@@ -42,7 +44,7 @@ module.exports = {
         ],
 
         messages: {
-            unexpectedComment: "Unexpected '{{matchedTerm}}' comment."
+            unexpectedComment: "Unexpected '{{type}}' comment: {{comment}}."
```

Comment:
I think we usually add `''` quotes in the message. Like `'{{type}}'`, it could be `'{{comment}}'`?